applications/nrf5340_audio/src/audio/Kconfig.defaults

```diff
@@ -35,6 +35,13 @@ config LC3_DEC_CHAN_MAX
 	int
 	default 1
 
+if TRANSPORT_BIS # Depends on TRANSPORT_BIS
+
+config AUDIO_TEST_TONE
+	default n
```

Comment:
No need to have this in the Kconfig.defaults. Just add 'default n if TRANSPORT_BIS' in the Kconfig